subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -163,21 +164,23 @@ int lll_init(void)
 	hal_swi_init();
 
 	/* Connect ISRs */
-	IRQ_DIRECT_CONNECT(RADIO_IRQn, CONFIG_BT_CTLR_LLL_PRIO,
-			   radio_nrf5_isr, IRQ_CONNECT_FLAGS);
+	NRF_IRQ_DIRECT_DYNAMIC(RADIO_IRQn, CONFIG_BT_CTLR_LLL_PRIO,
```

Comment:
Actually can you please gate all this behind a Kconfig option? Many small apps don't need it and it increases the footprint in both ROM and RAM. Whenever an app needs to shut down bluetooth it can enable this option. It can even be the same Kconfig option that we can use later in the Host: `CONFIG_BT_ENABLE_DISABLE` ;)

Reply:
@carlescufi This is only added here to illustrate how it could be used. I will take this commit out and do it properly later. I`ll include your suggestion for a Kconfig option then.